admin_scripts/ubuntu/ubuntu_16.04/Upgrade_LibreOffice_5.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+add-apt-repository ppa:libreoffice/ppa
+apt-get update
+apt dist-upgrade
```

Comment:
I'm not sure if this script is still current, but if it is, is `dist-upgrade` really what we want? Isn't it enough to install the relevant updated LibreOffice packages?

(Also, since this is an `apt` script, we'll need `export DEBIAN_FRONTEND=noninteractive` at the top somewhere. Hmm... maybe we should just make sure that `bibos_client` sets that variable itself before running any scripts? There are no reasons why OS2borgerPC scripts should ever ask for user input on packages...)